layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,208 @@
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li>and countless more names not listed here whose lives were taken by violence and brutality</li>
+    </ul>
+    <p><a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">too</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">many</a> more.</p>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+    <p>In Node.js, we're going to be working on supporting Black members of the community. If we can help - by using our collective platforms to amplify your words, by helping you in your career or in contributing to open-source, or in any other way - we are here for you.</p>
+
+    <!-- <section id="we-are-here-for-you">Tierney Cyren (<a href='https://twitter.com/bitandbang/status/1267893948149911556?s=20'>message of support</a>, <a href='https://twitter.com/bitandbang'>twitter</a>)</section> -->
+
+    <section id="emails">Please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively.</section>
+  </section>
+
+  <section id="temporary">
+    If you're looking for docs, they're available in <a href="https://github.com/nodejs/node/tree/master/doc">nodejs/node</a>. If you're looking for the most recent security release, please see the temporary <a href="https://nodejs.org/en/blog/vulnerability/june-2020-security-releases/">security release landing page</a> in lieu of the downloads page and blog post.
```

Comment:
In my humble opinion, other docs are still alive, and some people would like to see the API pages and download pages, so we would be better to show more better links like below. 

ex) `<a href="https://nodejs.org/en/docs/">nodejs.org docs</a>`

Reply:
all the links will still be alive and reachable... just not from the home page. Part of not including them right now is intended to be part of the action.

Reply:
I see.

But in my opinion, I think various people would be better to have various styles to tackle this situation. 
If an application developer would like to create some apps to save black community using node.js, reachable link would be useful.

FYI: React page has a global header and show the statement on top, Angular is also same.
https://reactjs.org/
https://angular.io/

I really feel this action is important. But I am Asian people, maybe I don't understand deeply how serious is this situation. If my opinion is bad, feel free to ignore.

Reply:
> Not being able to find a link on a page is not a particularly burdensome challenge 

https://github.com/nodejs/nodejs.org/pull/3202#issuecomment-638360890

I think so too. I don't want to be a blocker about this PR. I think we would be better to go forward and land this statement quickly. if Japanese node community needs docs and links, I will follow them on SNS / blog / github etc.